routes/web.php

```diff
@@ -102,3 +107,13 @@
     ->name('shoppingcart.insert');
 Route::post('/shoppingcart/update', [ShoppingCartController::class, 'update'])
     ->name('shoppingcart.update');
+
+// track orders
+Route::middleware('auth')->group(function () {
+    Route::get(__('route.track_orders'), [OrderTrackingController::class, 'index'])
+        ->name('track_orders.index');
+
+    Route::get('bestellingen-volgen/{order}', [OrderTrackingController::class, 'details'])
+        ->name('track_orders.details');
+});
```

Comment:
Bovenaan deze klasse wordt deze middleware al gebruikt, daar kan je het in zetten